sdk/schemaregistry/Azure.Data.SchemaRegistry/tests/Samples/SchemaRegistryReadmeSnippets.cs

```diff
@@ -54,11 +52,11 @@ public void RegisterSchema()
 
         [Ignore("Only verifying that the sample builds")]
         [Test]
-        public void GetSchemaId()
+        public void RetrieveSchemaId()
```

Comment:
Should try to use the naming convention of Sample(number)_(description), e.g. Sample01_HelloWorld, Sample02_OtherStuff..

Reply:
The template had this naming convention: `MiniSecretClientSamples.HelloWorld`. But, I've updated these files to use the underscore convention above.